### ISSUE TITLE
Why not use JWT for authentication?

### ISSUE THREAD

**Issue by qooco** [theodorejb/phasem#2]
Why custom db AuthTokens, not JWT ?
How about email verification?

**Comment by theodorejb** [comment]
> Why custom db AuthTokens, not JWT ?

I've used JWT on other projects in the past, and if I can help it I won't use it again. The standard is fundamentally flawed in several ways (see https://paragonie.com/blog/2017/03/jwt-json-web-tokens-is-bad-standard-that-everyone-should-avoid). Also JWT requires maintaining a single secret, and if that secret is ever compromised a hacker could impersonate any of your users.

The approach I used for Phasem doesn't rely on a single secret, and even if someone reads the auth tokens in the database they can't impersonate users.

> How about email verification?

I haven't had time to implement this.
